packages/block-library/src/rss/style.scss

```
@@ -41,3 +33,10 @@ ul.wp-block-rss { // The ul is needed for specificity to override the reset styl
 	display: block;
 	font-size: 0.8125em;
 }
+// This block has customizable padding, border-box makes that more predictable.
+// This needs extra specificity due to the reset mixin on the parent: https://github.com/WordPress/gutenberg/blob/a250e9e5fe00dd5195624f96a3d924e7078951c3/packages/edit-post/src/style.scss#L54
+.wp-block-rss {
+	box-sizing: border-box;
+	list-style: none;
+	padding: 0;
+}
```

```suggestion
.wp-block-rss {
	// This block has customizable padding, border-box makes that more predictable.
	box-sizing: border-box;
	list-style: none;
	padding: 0;
}
```

We should move the border-box comment next to the rule it relates to. The other comment about specificity no longer applies as this style is using the simple `0-1-0` selector.